docs/UserGuide.md

```diff
@@ -336,9 +336,9 @@ For detailed explanations of each flag and acceptable arguments, refer to Sectio
   ```
   add n/ JOHN DOE p/ 99007766 e/ mrdoe@ntu.sg a/ com3 j/ doctor i/ 99999
   ```
-- Add new client with tier and remark:
+- Add new client with tier and remark(s):
   ```
-  add n/ JOHN DOE p/ 99007766 e/ mrdoe@ntu.sg a/ com3 j/ doctor i/ 99999 t/ gold rn/ got anger issue
+  add n/ JOHN DOE p/ 99007766 e/ mrdoe@ntu.sg a/ com3 j/ doctor i/ 99999 t/ gold r/ got anger issue
```

Comment:
just for clarity's sake, should we add a "s/ urgent" here to show all the prefixes users can use?

Reply:
Good idea! I'll implement it